quickwit/Cargo.toml

```diff
@@ -50,7 +50,7 @@ backoff = { version = "0.4", features = ["tokio"] }
 base64 = "0.21"
 byte-unit = { version = "4", default-features = false, features = ["serde"] }
 bytes = "1"
-chitchat = { git = "https://github.com/quickwit-oss/chitchat", rev = "4973853" }
+chitchat = { path = "../../chitchat/chitchat" }
```

Comment:
this needs to point to a rev within main before merging.

Reply:
We're almost there on the Chitchat front, there's one more PR to be merged, and then we'll be good to go.